portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase

```diff
@@ -88,9 +88,24 @@ definition {
 	@description = "Verify that the Workspace label is present when a client extension is created through osgi"
 	@priority = 3
 	test OsgiClientExtensionWorkspaceLabelIsPresent {
+		task ("Given config file for client extension of type custom element created") {
+			ClientExtension.createGlobalVariableClientExtensionType(type = "customElement");
+
+			ClientExtension.checkConfigFileCreatedProperly();
```

Comment:
Remove this line. We don't want assertions in Given and When steps, we only want to make validation checks on what we actually want to test in the Then steps.